src/ext/exe.rs

```diff
@@ -621,12 +624,7 @@ trait Command {
     fn github_owner(&self) -> &str;
     fn github_repo(&self) -> &str;
     fn download_url(&self, target_os: &str, target_arch: &str, version: &str) -> Result<String>;
-    fn executable_name(
-        &self,
-        target_os: &str,
-        target_arch: &str,
-        version: Option<&str>,
-    ) -> Result<String>;
+    fn executable_name(&self, target_os: &str, target_arch: &str, version: &str) -> Result<String>;
```

Comment:
Made `version` a `&str` instead of an `Option<&str>` since the only place the function is called, the version is guarunteed to be known. It also doesn't seem like this function will be called in a different place.